apis/common/v1/resource.go

```diff
@@ -203,6 +203,11 @@ type ResourceSpec struct {
 	// Deprecated: Please use ProviderConfigReference, i.e. `providerConfigRef`
 	ProviderReference *Reference `json:"providerRef,omitempty"`
 
+	// ManagementPolicy specifies how this managed resource should be managed.
+	// +optional
+	// +kubebuilder:default=FullControl
+	ManagementPolicy ManagementPolicy `json:"managementPolicy,omitempty"`
```

Comment:
Let's mark this field as alpha. In core APIs we've done this by including the following text in the doctoring:

```go
// THIS IS AN ALPHA FIELD. Do not use it in production. It is not honored
// unless the relevant Crossplane feature flag is enabled, and may be
// changed or removed without notice.
```

I think it would also be worth explaining the interplay between `ManagementPolicy` and `DeletionPolicy` in one or both of the field comments.